Modules/IO/NIFTI/src/itkNiftiImageIO.cxx

```diff
@@ -1753,21 +1753,26 @@ Normalize(std::vector<double> & x)
 static bool
 IsAffine(const mat44 & nifti_mat)
 {
-  const vnl_matrix_fixed<float, 4, 4> mat(&(nifti_mat.m[0][0]));
+  vnl_matrix_fixed<double, 4, 4> mat;
+
+  for (int i = 0; i < 4; i++)
+    for (int j = 0; j < 4; j++)
```

Comment:
Instead of `i++` and `j++`, please write `++i` and `++j`, following pull request #2499 commit 971aef211cad336ba75cb2da114fc86e873444e5 "STYLE: Replace postfix by prefix increment in `for` loops in ITK/Modules"